gcn/notices/chime/frb.retraction.example.json

```
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/chime/frb.schema.json",
+  "alert_type": "retraction",
+  "id": ["FRB 20231013B"],
+  "trigger_time": "2024-09-18T07:19:10.765268Z",
+  "trigger_time_error": [0.007864319719374001],
```

```suggestion
  "trigger_time_error": 0.007864319719374001,
```
Now, numbers are also allowed for errors.